### PR TITLE
Typecode conflict detection

### PR DESCRIPTION
**Pull request by klaussfreire**
Try to detect conflicts between user-registered schemas and
builtin typecodes

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -7,13 +7,25 @@ The format is largely inspired by keepachangelog_.
...
+  of subtle breakage.
+
 v0.6.2 - 2019-02-21
 ===================
 
 Bugfixes
 --------
 
-- Map only requested size, when specified, in id mappers.
+- Disallow registering schemas with typecodes that conflict
```
Comment (tzulberti-jampp) [inline CHANGELOG.rst]: Why did you remove the original message?

Comment (klaussfreire) [reply CHANGELOG.rst]: Bad merge

**Review by tzulberti-jampp — APPROVED** [review]

**Review by gefarion — APPROVED** [review]
